### PR TITLE
[new] add le_js@0.0.3

### PR DESCRIPTION
**Pull request by Amomo**
Hi @maruilian11

For #6258, I add le_js at v0.0.3 and add git auto-update config in its package.json.
repo url: https://github.com/logentries/le_js

Though this repo seems don't have watch/star over 100 on GitHub.
At the bottom of its [HomePage](https://logentries.com/) said that it has more than 35K users.

Would you mind checking this PR for me? Thank you~ :-D

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : **Watch `22`, Star `49`, Fork `33`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

### CODE REVIEW

File: ajax/libs/le_js/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "le_js",
+  "filename": "le.min.js",
+  "version": "0.0.3",
+  "description": "Javascript library for Logentries.com",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/logentries/le_js.git"
+  },
+  "author": "",
```
Comment (PeterDaveHello) [inline ajax/libs/le_js/package.json]: Why to use empty value?

Comment (Amomo) [inline ajax/libs/le_js/package.json]: The original package.json is empty, maybe I should remove this field?

Comment (PeterDaveHello) [inline ajax/libs/le_js/package.json]: Should add it.

Comment (Amomo) [inline ajax/libs/le_js/package.json]: I have added their organization name and email address.

**Comment by maruilian11** [comment]
@PeterDaveHello i think this PR is ok.

**Comment by PeterDaveHello** [comment]
what about ls_js.dyn.js ?

**Comment by maruilian11** [comment]
@PeterDaveHello readme.md in product folder shows
```>
Several files are produced in the build. You'll probably just want to use le.min.js, however.
```
so i think we only need le.js and its min file

**Comment by PeterDaveHello** [comment]
https://github.com/logentries/le_js/tree/master/product#contents

**Comment by maruilian11** [comment]
@PeterDaveHello i think this pr is ok.

**Pull request merged**